packages/image-block/src/components/Image.tsx

```diff
@@ -24,8 +25,12 @@ export const ImageComponent = ({ image, blockSettings, isEditing, appBridge }: I
     const { open } = useAssetViewer(appBridge);
     const link = blockSettings?.hasLink && blockSettings?.linkObject?.link && blockSettings?.linkObject;
     const imageStyle = getImageStyle(blockSettings, image.width);
+    const { assetViewerEnabled: globalAssetViewerEnabled } = usePrivacySettings(appBridge);
```

Comment:
This hook is now used twice, can you use it just once and pass the assetViewEnabled just as a prop?

Reply:
Adjusted